go.mod

```diff
@@ -1,6 +1,6 @@
 module k8s.io/registry.k8s.io
 
-go 1.18
+go 1.19
```

Comment:
we don't care about supporting older go, this repo is only used by us and built with the latest go.